tests/testsuite/package.rs

```diff
@@ -7807,3 +7807,33 @@ fn publish_to_alt_registry_warns() {
 "#]])
         .run();
 }
+
+#[cargo_test]
+fn package_dir_not_excluded_from_backups() {
+    // This test documents the current behavior where target directory is NOT excluded from backups.
+    // After the fix, this test will be updated to verify that CACHEDIR.TAG exists.
+    let p = project()
+        .file("src/main.rs", r#"fn main() { println!("hello"); }"#)
```

Comment:
nit: for faster test execution (avoid to binary linking)

```suggestion
        .file("src/lib.rs", "")
```

Reply:
Thanks for the tip! Done.